### PR TITLE
Remove alpha reference from PersistentVolumeStatus.lastPhaseTransitionTime docs

### PR DESCRIPTION
**Pull request by RomanBednar**
#### What type of PR is this?
/kind documentation
/kind cleanup

#### What this PR does / why we need it:

`lastPhaseTransitionTime` field is no longer alpha, remove the statement from comments.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: pkg/apis/core/types.go [hunk 1]
```diff
@@ -392,7 +392,6 @@ type PersistentVolumeStatus struct {
 	Reason string
 	// LastPhaseTransitionTime is the time the phase transitioned from one to another
 	// and automatically resets to current time everytime a volume phase transitions.
-	// This is an alpha field and requires enabling PersistentVolumeLastPhaseTransitionTime feature.
```
Comment (liggitt) [inline pkg/apis/core/types.go:395]: change to `This is a beta field...`

and update the godoc on the feature gate to record the release where it graduated:

```
	// owner: @RomanBednar
	// kep: https://kep.k8s.io/3762
	// alpha: v1.28
	//
	// Adds a new field to persistent volumes which holds a timestamp of when the volume last transitioned its phase.
	PersistentVolumeLastPhaseTransitionTime featuregate.Feature = "PersistentVolumeLastPhaseTransitionTime"
```

Comment (RomanBednar) [reply pkg/apis/core/types.go:395]: Done.

**Comment by liggitt** [comment]
lgtm, but looks like one more doc file needs regeneration (see the verify failure)

**Pull request merged**